### PR TITLE
Working on excercise 4

### PR DESCRIPTION
**Pull request by AnaCDomingo**
I'm having a lot of trouble with this topic! D:

### CODE REVIEW

File: 01-JS-DOM-APIs/Ex04/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <div class="repWrapper">
+      <h2>Repositories:</h2>
+      <ul id="repoList">
+      </ul>
+    </div>
+    <script>
+      function getRep() {
+        let repRequest = new XMLHttpRequest();
+        repRequest.open ('GET', 'https://api.github.com/search/repositories?q=JavaScript', true);
+        repRequest.send ();
```
Comment (mtulian) [inline 01-JS-DOM-APIs/Ex04/index.html]: Let's hold on this one. Once you finished the exercise 3, we go a head an review the 4. You will need to add the reusable function to make Ajax calls here.

**Review by mtulian — APPROVED** [review]
Looks fine, let's merge and start working in topic 2

**Comment by AnaCDomingo** [comment]
What about the failed XHR request? I couldn't figure where the mistake was.

File: 01-JS-DOM-APIs/Ex04/index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <div class="repWrapper">
+      <h2>Repositories:</h2>
+      <ul id="repoList">
+      </ul>
+    </div>
+    <div class="searchBox">
+      <form name="search">
+        <label>Search a repository:</label>
+        <input type="text" id="repoSearch">
+        <input type="submit" value="Search" onclick="qSearch(document.getElementById('repoSearch').value)">
```
Comment (mtulian) [inline 01-JS-DOM-APIs/Ex04/index.html:26]: You can use the parameter in javascript directly, there is no need to provide it here. Did you tried that?

**Review by mtulian — APPROVED** [review]
It's fine!

**Pull request merged**